zep-memory-assistant/app.py

```
@@ -136,6 +137,9 @@ def handle_conversations(agent, user, prompt):
     with st.chat_message("user"):
         st.markdown(prompt)
 
+    # Append /no_think token for the backend processing
+    prompt_with_token = f"{prompt} /no_think"
```

Ruff (0.8.2)

Local variable `prompt_with_token` is assigned to but never used

Remove assignment to unused variable `prompt_with_token`

(F841)